### PR TITLE
Unified configuration for chaos controller

### PR DESCRIPTION
**Pull request by Gallardot**
### What problem does this PR solve? 
#598 legacy, Unified configuration for chaos controller
### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: cmd/controller-manager/main.go [hunk 1]
```diff
@@ -69,39 +61,26 @@ func init() {
...
 func main() {
 	parseFlags()
-
 	version.PrintVersionInfo("Controller manager")
 	if printVersion {
 		os.Exit(0)
 	}
 
+	// set RPCTimeout config
+	utils.RPCTimeout = common.ControllerCfg.RPCTimeout
```
Comment (Gallardot) [inline cmd/controller-manager/main.go:76]: I don't know what to do with this configuration item. It looks strange.

File: controllers/common/types.go [hunk 2]
```diff
@@ -38,15 +39,13 @@ const (
...
-var log = ctrl.Log.WithName("controller")
-
 //ControllerCfg is a global variable to keep the configuration for Chaos Controller
 var ControllerCfg *config.ChaosControllerConfig
 
 func init() {
 	conf, err := config.EnvironChaosController()
 	if err != nil {
-		log.Error(err, "Chaos Controller: invalid environment configuration")
+		fmt.Println(fmt.Sprintf("Chaos Controller: invalid environment configuration. err: %s", err))
```
Comment (yeya24) [inline controllers/common/types.go:48]: There is a redundant `fmt.Sprintf` here...

File: controllers/common/types.go [hunk 3]
```diff
@@ -38,15 +39,13 @@ const (
...
-
 //ControllerCfg is a global variable to keep the configuration for Chaos Controller
 var ControllerCfg *config.ChaosControllerConfig
 
 func init() {
 	conf, err := config.EnvironChaosController()
 	if err != nil {
-		log.Error(err, "Chaos Controller: invalid environment configuration")
+		fmt.Println("Chaos Controller: invalid environment configuration. err:", err)
 		os.Exit(1)
```
Comment (Yisaer) [inline controllers/common/types.go:49]: Using `fmt.Println` here is strange, I suggest to use `log.Fatal` here.

**Pull request merged**